app/presenters/notifications/setup/returns-period.presenter.js

```diff
@@ -18,9 +18,44 @@ function go() {
 }
 
 function _returnsPeriod() {
+  const today = new Date()
+  const currentYear = today.getFullYear()
+  const previousYear = currentYear - 1
+
+  if (_dayIsInJanuary(today)) {
+    return _dayIsInJanuaryOptions(currentYear, previousYear)
+  }
+
   return []
 }
 
+/*
+ *  Checks if a date is in January
+ *  A date is in January if it is between 1st January - 28th January
+ */
+function _dayIsInJanuary(date) {
+  return date.getMonth() === 0 && date.getDate() >= 1 && date.getDate() <= 28
```

Comment:
Really, _really_ minor NIT. But do we need to check `date.getDate() >= 1` _and_ `date.getDate() <= 28`. Wouldn't that last check suffice?

Reply:
Good point. can remove.